src/command-line/install.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const colors = require("colors/safe");
+const program = require("commander");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("install <package>")
+	.description("Install a package")
+	.on("--help", Utils.extraHelp)
+	.action(function(packageName) {
+		const fs = require("fs");
+		const fsextra = require("fs-extra");
+		const child = require("child_process");
+		const packageJson = require("package-json");
+
+		if (!fs.existsSync(Helper.CONFIG_PATH)) {
+			log.error(`${Helper.CONFIG_PATH} does not exist.`);
+			return;
+		}
+
+		packageJson(packageName, {
+			fullMetadata: true
+		}).then((json) => {
+			if (!("lounge" in json)) {
+				log.error(`${colors.red(packageName)} does not have The Lounge metadata.`);
+
+				process.exit(1);
+			}
+
+			log.info(`Installing ${colors.green(packageName)}...`);
+
+			const packagesPath = Helper.getPackagesPath();
+
+			// Create node_modules folder, otherwise npm will start walking upwards to find one
+			fsextra.ensureDirSync(packagesPath);
+
+			const npm = child.spawn(
+				process.platform === "win32" ? "npm.cmd" : "npm",
+				[
+					"install",
+					"--production",
+					"--no-save",
+					"--no-bin-links",
+					"--no-package-lock",
+					"--prefix",
+					packagesPath.replace(/node_modules$/, ""),
+					packageName
+				]
+			);
+
+			npm.on("error", (e) => {
+				log.error(`${e}`);
+				process.exit(1);
+			});
```

Comment:
On the channel earlier, when you said "I hid npm stdout", I didn't realize it meant that you simply don't redirect it (actually, for the warnings we could see, it's from `stderr`). I thought you were actively silencing the output or something, hence my suggestion.

If we ever need a clean `stderr` (for example if this block doesn't report everything it should), I checked and having a `package.json` with only `{ "private": true }` in it silences all the warnings :man_shrugging: 

Do you know what's the difference between `npm.on("error", ...)`  and `npm.stderr.on("data", ...)`? Warnings are sent to the latter, and installing a package that doesn't exist goes through your `catch`, but I'm not sure what this one does.

Reply:
I reverted hiding of stdout, and create package.json now.

Reply:
Oh no, can we please not do this. We can check whether the error code is 0 and if not we can print out the stdout, but creating a package.json when there isn't a package feels like a horrible solution.

Reply:
`package.json` is in `.lounge/packages/` folder, so there's just one for all the packages (not for each).

Reply:
I get that, but it's still misleading because packages/ isn't a node package in itself, so shouldn't have a package.json.

Reply:
Why does it matter?

Reply:
Why is it better than muting stderr? I'm not keen on having a file hanging around that is unnecessary (and could be misleading). Especailly one that says "private:true" which makes no sense for the packages dir. 

Realistically, does it matter? No. But I don't see why this is better than muting stderr. I won't stand in the way of the PR because of this but I don't like it.

Reply:
I don't like having `node_modules` folder for example, but since this is the easiest way of doing it with npm, we kept it. Ideally we would just have `packages/lounge-theme-xxx` and no extra stuff around it, and it would just work.

But since we chose to use npm in such a way, we have to deal with consequences.

Reply:
I agree with @xPaw, embracing the npm way is probably better on the long run than trying to go against it 😅

Also having a `package.json` might be a good thing, I can already see one thing or 2 that this could allow us in the future.

Reply:
I disagree, but fine, I won't stand in the way. I really would rather we just muted the output from the npm command, and log it if we need. 

> embracing the npm way

But this isn't the npm way, is it? This is us hacking npm to use in our own way. Npm is designed for packages in your own package (which this is not) or global modules for running commands (also this is not), so that's not an argument.

But as I say, I won't stand in the way if you both think this is the better way to do it.

Reply:
Let's keep the output for now, if it's annoying we can remove it in the future.